docs/community/tutorials-contest/2021/red-dev-avalanche-chainlink-tutorial/README.md

```diff
@@ -204,7 +204,7 @@ DATABASE_URL=postgresql://$USERNAME:$PASSWORD@$HOST:5432/chainlink?sslmode=disab
 Create a .api file to expose credentials for the API and GUI interfaces:
 ```
 $ echo "youremailaddress@yourcompany.com
-$PASSWORD" > ~/.chainlink-avalanche/.env
+$PASSWORD" > ~/.chainlink-avalanche/.api
```

Comment:
Actually it should be the opposite. All instances of `.api` should be changed to `.env`. `.api` is a photoshop format.